tutorials/advanced_features/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<ul><li>"... we will go into more detail."</li><li>"The State Preparation And Measurement (SPAM) errors encapsulate, as the name indicates, the errors that can occur during state preparation and measurements. They can be:"</li><li>A state preparation error (with probability&nbsp;η for each atom), where an atom is not correctly pumped to the ground state at the beginning, making it effectively unavailable throughout the sequence.</li><li>Measurement errors (with probabilities ...), which are linked to the incorrect identification of the atoms final state. During the imaging process, ..." </li><li>"On the contrary, atoms in |g&gt; might be kicked out for various reasons (e.g. collisions with residual gas in the chamber)..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>